src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -736,8 +739,9 @@ private static ClassLoader contextClassLoader(Thread parent) {
      *
      * @param name thread name, can be null
      * @param characteristics thread characteristics
+     * @param bound true when bound to an OS thread
```

Comment:
Nit: s/OS/VM/ ?

Reply:
I think it would be confusing to use "VM" here. The intention is that "true" means the virtual thread will be bound to an OS thread once it has started. Yes, technically it's whatever the VM implements but I think that is too much to try to explain in the parameter description.